conda_build/templates/cookie/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: {{ name }}
+{% raw %}
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}
```

Comment:
Please do not use this style of Jinja2 variables. Setting defaults kind of breaks the iterative parsing.  See https://github.com/conda/conda-build/issues/1646 for more info.